src/plugins/region_map/server/routes/opensearch.ts

```diff
@@ -5,6 +5,7 @@
 
 import { schema } from '@osd/config-schema';
 import { IRouter } from 'opensearch-dashboards/server';
+import { DEFAULT_INDEX_FETCH_SIZE } from '../../common/constants/shared';
```

Comment:
`../../common` should be exporting this and then the constant should be imported the other ones:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d9dc91de7e5c36625bb2ea01ba9453446f0307e9/src/plugins/region_map/public/components/map_choice_options.tsx#L24